### DISCUSSION TITLE
Following release question

### DISCUSSION THREAD

**Discussion by alitvak69** [containers/podman#11059] [General]
Hello everyone,

Looking at the number of changes after 3.2.3, I wonder if it makes sense to upgrade from 3.2.2 to 3.2.3  or wait for 3.2.4?  Or is it going to be 3.3.0?  Do you have any ETA for the new release regardless of the version?

**Comment by vrothberg** [comment]
Thank you for reaching out, @alitvak69!

I recommend updating from 3.2.2 to 3.2.3.  There is a number of important fixes.

The next version is likely going to be 3.3.0.  We plan to cut a first release candidate soon; probably by the end of this week or early next week.
